src/Ownable.ts

```diff
@@ -0,0 +1,42 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+
+export class Ownable {
+  hre: HardhatRuntimeEnvironment
+
+  constructor(hre: HardhatRuntimeEnvironment) {
+    this.hre = hre
+  }
+
+  /**
+   * Transfers ownership of an Ownable contract to a specific address.
+   *
+   * @param {string} contractName Deployed Contract Name
+   * @param {string} newOwnerAddress New Owner Address
+   * @param {from} from Address to send transaction from.
```

Comment:
Should be `string` instead of `from`.

Reply:
fa5c8488edba0c6dba0423d0848bbc5e0c117b52